CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### Bug fixes
 
 * [#541](https://github.com/rubocop/rubocop-rails/issues/541): Fix an error for `Rails/HasManyOrHasOneDependent` when using lambda argument and specifying `:dependent` strategy. ([@koic][])
+* [#547](https://github.com/rubocop/rubocop-rails/pull/547): Add new `Rails/ActionOrder` cop. ([@mollerhoj][])
```

Comment:
This needs to be under a `### New features` section below `## master` above (2.12.2 is already released), and you need to add a link for your github to the bottom of the file.